src/plugins/audioPlugin.js

```diff
@@ -20,15 +31,104 @@ export default function createAudioPlugin() {
     };
   };
 
-  const embedToHTML = embed => {
-    const { audio: { title, audioFile: { mimeType, url } } } = embed;
-    return `<figure class="article_audio"><audio controls type="${mimeType}" src="${url}"></audio><figcaption>${
-      title.title
-    }</figcaption></figure>`;
+  const onError = ({ audio }, locale) => {
+    const { audioFile: { mimeType, url } = {} } = audio || {};
+    return renderToStaticMarkup(
+      <Figure>
+        {audio ? (
+          <AudioPlayer type={mimeType} src={url} />
+        ) : (
+          <AudioPlayer disabled />
```

Comment:
Siden vi ikke har noe disabled støtte forelsår jeg å retunere en svg istedenfor. F.eks. en variant av den som blir brukt for bilder:

 ```
    <svg
          fill="#8A8888"
          height="50"
          viewBox="0 0 24 12"
          width="100%"
          xmlns="http://www.w3.org/2000/svg"
          style={{ backgroundColor: '#EFF0F2', display: 'block' }}>
          <path d="M0 0h24v24H0V0z" fill="none" />
          <path
            transform="scale(0.3) translate(28, 8.5)"
            d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
```

Reply:
Ok, det jeg tenkte var at vi kunne sette inn en "greyed out" lydspiller for å ikke endre designet, men har lagt inn svgen du foreslo.